drivers/counter/counter_dtmr_cmsdk_apb.c

```diff
@@ -75,6 +94,9 @@ static const struct counter_driver_api counter_dtmr_cmsdk_apb_api = {
 	.stop = counter_dtmr_cmsdk_apb_stop,
 	.read = counter_dtmr_cmsdk_apb_read,
 	.set_alarm = counter_dtmr_cmsdk_apb_set_alarm,
+	.disable_alarm = counter_dtmr_cmsdk_apb_disable_alarm,
+	.set_wrap = counter_dtmr_cmsdk_apb_set_wrap_alarm,
```

Comment:
please align to latest changes `set_wrap` -> `set_top_value`, `get_wrap`->`get_top_value`

Reply:
The fact that CI is still passing suggests that we're not building this driver in CI. @fvincenzo 	will you please add `counter` to the supported list in the appropriate \<board\>.yaml file? Like this: https://github.com/zephyrproject-rtos/zephyr/blob/817c74fbd0546f10d5ff5f66d5a1aeeb09f8b575/boards/arm/frdm_kw41z/frdm_kw41z.yaml#L9-L12